index.html

```diff
@@ -12,8 +12,24 @@
     <header>
       <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
+        <i class="fa fa-bars"></i>
       </button>
+      <nav id="mobile-nav" class="d-none">
+        <button type="button" id="header-close-menu">
+            <i class="fa fa-times"></i>
```

Comment:
- [ ] As I mentioned before, take care of the typos. Here you are missing another `s`, so you should have `fas fa-times` instead of `fa fa-times`. In this case, it's about a class name, but any typo can produce bigger bugs, so stay alert! 🙌